script/strategies/aave-v3/arbitrum/DeployStrategies.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import { BaseDeployStrategies, IAToken } from "../BaseDeployStrategies.sol";
+import { DeployPeriphery } from "script/DeployPeriphery.sol";
+
+contract DeployStrategies is DeployPeriphery, BaseDeployStrategies {
+  function run() external override(DeployPeriphery) {
+    vm.startBroadcast();
+    deployPeriphery();
+    deployAaveV3Strategies();
+    vm.stopBroadcast();
+  }
+
+  function deployAaveV3Strategies() internal {
```

Comment:
Let's take the guardians and judges arrays as params, I think that will help us in the future